support/provenance/provenance.py

```diff
@@ -92,6 +133,9 @@ def get_historic(provenance: {str: str}, reset: bool) -> {str: str}:  # TODO: po
 
     log.info(
         f'found {len(history)} products needing update of a {count} full history of {len(provenance)} total products')
+    if log.isEnabledFor(logging.DEBUG):
+        for lidvid in history.keys():
+            log.debug(f'{lidvid}')
```

Comment:
We might be able to have a single line for that, if we don't need a single debug message for each lidvid

log.debug(', '.join(history.keys()))

or 

log.debug('\n'.join(history.keys()))

i have not tested that though